src/components/ui/search-input/SearchInputWithTags.test.ts

```
@@ -0,0 +1,168 @@
+        suggestions: SUGGESTIONS,
+        ...props
+      }
+    })
+
+    return { ...result, user }
+  }
+
+  describe('icon states', () => {
+    it('shows search icon when empty', () => {
+      renderComponent()
+      const icon = document.querySelector('.icon-\\[lucide--search\\]')
```

ESLint

[error] Avoid direct Node access. Prefer using the methods from Testing Library.

(testing-library/no-node-access)